### PR TITLE
Fix callout drawing that happens even if disabled

### LINKED ISSUE #681: Label callout is drawn even if not enabled
Label callout is drawn even if not enabled

### PR DESCRIPTION
**Pull request by stockiNail**
Fix #681

### CODE REVIEW

File: src/types/label.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ function calculatePosition(start, size, adjust = 0, position) {
 
 function drawCallout(ctx, element) {
   const {pointX, pointY, calloutPosition, options} = element;
-  if (element.inRange(pointX, pointY)) {
+  if (element.inRange(pointX, pointY) || !calloutPosition) {
```
Comment (kurkle) [inline src/types/label.js:130]: Turn the checks around, `!calloutPosition` is a faster test and the inRange test is skipped when that is true.

Comment (kurkle) [reply src/types/label.js:130]: + add a test

Comment (stockiNail) [reply src/types/label.js:130]: YES!

Comment (stockiNail) [reply src/types/label.js]: done

**Comment by kurkle** [comment]
Some kind of regression test would be good to have, as this went through. A simple fixture is probably best, maybe as additional annotation in an existing one testing callouts?

**Comment by stockiNail** [comment]
> Some kind of regression test would be good to have, as this went through. A simple fixture is probably best, maybe as additional annotation in an existing one testing callouts?

Yes, I'll commit it in next minutes.

**Review by kurkle — APPROVED** [review]

**Pull request merged**